src/oci-umount.c

```diff
@@ -580,6 +580,18 @@ static int parseBundle(yajl_val *node_ptr, char **rootfs, struct config_mount_in
 	}
 	*rootfs=strdup(YAJL_GET_STRING(v_root));
 
+	/* Prepend bundle path if the rootfs string is relative */
+	if (*rootfs[0] != '/') {
+		char *new_rootfs;
+
+		asprintf(&new_rootfs, "%s/%s", YAJL_GET_STRING(v_bundle_path), *rootfs);
+		if (!new_rootfs) {
+			pr_perror("failed to alloc rootfs");
+			return EXIT_FAILURE;
+		}
+		*rootfs = new_rootfs;
```

Comment:
Oh Shoot!  👍 to jwessel.  I totally messed up. I though the definition was *rootfs, not **rootfs.  Then yes, the * is needed there.  In my testing I defined it as *rootfs and that's why I'd the error with the star in play.  Apologies for the churn.